tests/unit/Extension/Attributes/Util/AttributesHelperTest.php

```diff
@@ -58,6 +58,7 @@ public function dataForTestParseAttributes(): iterable
         yield [new Cursor('{: target=_blank}'), ['target' => '_blank']];
         yield [new Cursor('{: target=_blank }'), ['target' => '_blank']];
         yield [new Cursor('{:   target=_blank   }'), ['target' => '_blank']];
+        yield [new Cursor('{: disabled}'), ['disabled' => '']];
```

Comment:
I think we probably want the attribute value to be `true` so that `HtmlElement` renders just the attribute name:

https://github.com/thephpleague/commonmark/blob/5af699c47955345e14f66f3863ff7db48dcd90a2/src/Util/HtmlElement.php#L124-L132

Could we also edit the test on line 56 to include a single attribute?  (That test case proves that different styles of attributes can be used together)